harvard-university-for-the-creative-arts.csl

```diff
@@ -82,9 +90,22 @@
             <group delimiter=" ">
               <group delimiter=" ">
                 <text variable="title" font-style="italic" suffix="."/>
+                <choose>
+                  <if match="any" variable="volume">
+                    <text term="volume" form="short" text-case="capitalize-first"/>
+                    <text variable="volume" text-case="uppercase" suffix="."/>
+                  </if>
+                </choose>
                 <text macro="edition-no"/>
+                <group>
+                  <choose>
+                    <if match="any" variable="collection-title">
+                      <text term="in" text-case="capitalize-first" suffix=": "/>
+                      <text variable="collection-title" font-style="normal" suffix="."/>
+                    </if>
+                  </choose>
+                </group>
```

Comment:
Hi

Is it the conditional that I need to remove, if so is there a way of making the collection title only appear for a map without a conditional before it? That is what I am trying to achieve here, the map collection appearing at the end of the reference after the publisher